### ISSUE TITLE
UnhandledPromiseRejectionWarning

### ISSUE THREAD

**Issue by niftylettuce** [lassjs/lass#48]
```sh
(node:3252) UnhandledPromiseRejectionWarning: Error: Line: 4 Column: 60 - Unexpected identifier
    at Object.exports.parse (/Users/user/Projects/lass/node_modules/alce/lib/alce.js:20:11)
    at module.exports (/Users/user/Projects/lass/node_modules/fixpack/fixpack.js:49:19)
    at post (/Users/user/Projects/lass/sao.js:195:5)
    at kopy.then.stream (/Users/user/Projects/lass/node_modules/sao/lib/generate.js:226:40)
```

**Comment by niftylettuce** [comment]
This is due to having quotes in description that are unescaped.

**Comment by niftylettuce** [comment]
Released in v2.1.2, publishing to npm now
